### PR TITLE
Adding option to include additional java_opts

### PR DESCRIPTION
**Pull request by timdeluxe**
## Notes for Reviewers

- [ ] The commit history must be preserved - please use the rebase-merge or standard merge option instead of squash-merge
- [ ] Sync up with the author before merging

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -17,4 +17,5 @@
 
   $java_initial_heap_size = '1g'
   $java_max_heap_size = '1g'
+  $java_opts = '1g'
```
Comment (malcyon) [inline manifests/params.pp:20]: Shouldn't this default be an empty string?

**Review by malcyon — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! I only see one minor thing.

File: manifests/params.pp (hunk 1 above)
Comment (timdeluxe) [reply manifests/params.pp]: Sorry, you are right. I fixed it.

**Review by malcyon — APPROVED** [review]
I validated this on my local machine, and it looks good! Thanks so much!

**Comment by timdeluxe** [comment]
@malcyon Any module release planned here, which includes this merge?

**Comment by malcyon** [comment]
I'm not part of the Graylog team anymore. @bernd might be able to take a look at it.

**Pull request merged**